cmd/ebrelayer/relayer/relayer.go

```diff
@@ -32,8 +33,13 @@ import (
 // -------------------------------------------------------------------------
 
 func InitRelayer(cdc *amino.Codec, chainId string, provider string,
-								 contractAddress common.Address, eventSig string,
-								 validator sdk.AccAddress) error {
+	contractAddress common.Address, eventSig string,
+	validator sdk.AccAddress) error {
+
+	passphrase, err := keys.GetPassphrase("validator")
```

Comment:
Is this method of pulling the validator's key secure for multiple independent users? Can this be applied safely for local validators in addition to the public test "validator" account being queried here?

Reply:
You're reviewing an outdated file?